packages/web-react/src/components/Container/__tests__/Container.test.tsx

```diff
@@ -7,16 +7,30 @@ import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
 import Container from '../Container';
 
 describe('Container', () => {
+  const text = 'Hello world';
+  const testId = 'flex-test-id';
+
   classNamePrefixProviderTest(Container, 'Container');
 
   stylePropsTest(Container);
 
   restPropsTest(Container, 'div');
 
   it('should render text children', () => {
-    const dom = render(<Container>Hello World</Container>);
+    render(<Container data-testid={testId}>{text}</Container>);
+
+    expect(screen.getByText(text)).toBeInTheDocument();
```

Comment:
**suggestion (if-minor):** Can you please split the test? One assert for one test. You are testing `toBeInDocument` twice and IMHO this should be in a separate test.

Reply:
I copypasted this from some other component :D